AnagramSolver.BusinessLogic/Classes/Repositories/FileToDatabaseRepository.cs

```diff
@@ -0,0 +1,40 @@
+﻿using AnagramSolver.Contracts.Interfaces;
+using AnagramSolver.EF.DatabaseFirst.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AnagramSolver.BusinessLogic.Classes.Repositories
+{
+    public class FileToDatabaseRepository : IFileToDatabaseRepository
+    {
+        private readonly VocabularyDBContext _context;
+        public FileToDatabaseRepository(VocabularyDBContext context)
+        {
+            _context = context;
+        }
+        public async Task AddWordsToDatabase(HashSet<string> vocabulary)
+        {
+            try
+            {
+                foreach (var word in vocabulary)
+                {
+                    var wordToAdd = new Word { Word1 = word };
```

Comment:
This statement can be avoided in database layer. Consider moving this to Business layer, keeping repositories clean.